src/actions/index.js

```diff
@@ -234,3 +234,16 @@ export function toggleFocusMode(modularPipeline) {
     modularPipeline,
   };
 }
+
+export const TOGGLE_HOVERED_FOCUS_MODE = 'TOGGLE_HOVERED_FOCUS_MODE';
+
+/**
+ * Update the value of the of the currently-active hovered node from focus mode
+ * @param {string|null} nodeHovered The node's unique identifier
```

Comment:
```suggestion
 * @param {string|null} hoveredFocusMode The node's unique identifier
```

Reply:
Please check my last commit for this fix. Thanks